XIV on Mac/de.lproj/Main.strings

```diff
@@ -381,97 +381,97 @@
 
 /* Class = "NSMenu"; title = "Window"; ObjectID = "Td7-aD-5lo"; */
-"Td7-aD-5lo.title" = "Window";
+"Td7-aD-5lo.title" = "Fenster";
 
 /* Class = "NSTextFieldCell"; title = "Select an option:"; ObjectID = "TeW-Wy-5YR"; */
-"TeW-Wy-5YR.title" = "Select an option:";
+"TeW-Wy-5YR.title" = "Wähle eine Option aus:";
 
 /* Class = "NSSecureTextFieldCell"; placeholderString = "required"; ObjectID = "TfY-Li-gp6"; */
-"TfY-Li-gp6.placeholderString" = "required";
+"TfY-Li-gp6.placeholderString" = "Erforderlich";
 
 /* Class = "NSMenuItem"; title = "4.6.2"; ObjectID = "ThK-Bg-ZyU"; */
 "ThK-Bg-ZyU.title" = "4.6.2";
 
 /* Class = "NSMenuItem"; title = "Capitalize"; ObjectID = "UEZ-Bs-lqG"; */
-"UEZ-Bs-lqG.title" = "Capitalize";
+"UEZ-Bs-lqG.title" = "Großschreiben";
 
 /* Class = "NSTextFieldCell"; title = "Current Step"; ObjectID = "Uem-xO-XvC"; */
-"Uem-xO-XvC.title" = "Current Step";
+"Uem-xO-XvC.title" = "Aktueller Schritt";
 
 /* Class = "NSTextFieldCell"; title = "If Dalamud is not loading succesfully, you can try adjusting the load delay up or down to find a value that works for your Mac."; ObjectID = "VZh-e3-0dP"; */
-"VZh-e3-0dP.title" = "If Dalamud is not loading succesfully, you can try adjusting the load delay up or down to find a value that works for your Mac.";
+"VZh-e3-0dP.title" = "Wenn Dalamud nicht erfolgreich geladen wird, kannst du versuchen, die Ladeverzögerung nach oben oder unten zu verschieben, um einen Wert zu finden, der für deinen Mac geeignet ist.";
 
 /* Class = "NSMenuItem"; title = "Hide Others"; ObjectID = "Vdr-fp-XzO"; */
-"Vdr-fp-XzO.title" = "Hide Others";
+"Vdr-fp-XzO.title" = "Andere ausblenden";
 
 /* Class = "NSMenu"; title = "Edit"; ObjectID = "W48-6f-4Dl"; */
-"W48-6f-4Dl.title" = "Edit";
+"W48-6f-4Dl.title" = "Bearbeiten";
 
 /* Class = "NSMenuItem"; title = "Open Install Folder"; ObjectID = "Was-JA-tGl"; */
-"Was-JA-tGl.title" = "Open Install Folder";
+"Was-JA-tGl.title" = "Installationsordner öffnen";
 
 /* Class = "NSTextFieldCell"; title = "Shader Cache"; ObjectID = "Wum-vu-uC1"; */
 "Wum-vu-uC1.title" = "Shader Cache";
 
 /* Class = "NSTextFieldCell"; title = "Installing..."; ObjectID = "X9z-xO-PQz"; */
-"X9z-xO-PQz.title" = "Installing...";
+"X9z-xO-PQz.title" = "Installieren...";
 
 /* Class = "NSMenuItem"; title = "x86"; ObjectID = "XO2-yP-LkT"; */
 "XO2-yP-LkT.title" = "x86";
 
 /* Class = "NSMenuItem"; title = "Find…"; ObjectID = "Xz5-n4-O0W"; */
-"Xz5-n4-O0W.title" = "Find…";
+"Xz5-n4-O0W.title" = "Finde...";
 
 /* Class = "NSMenuItem"; title = "Find and Replace…"; ObjectID = "YEy-JH-Tfz"; */
-"YEy-JH-Tfz.title" = "Find and Replace…";
+"YEy-JH-Tfz.title" = "Suchen und Ersetzen...";
 
 /* Class = "NSButtonCell"; title = "Device Info"; ObjectID = "YOz-MM-jki"; */
-"YOz-MM-jki.title" = "Device Info";
+"YOz-MM-jki.title" = "Gerät Info";
 
 /* Class = "NSMenuItem"; title = "Start Speaking"; ObjectID = "Ynk-f8-cLZ"; */
-"Ynk-f8-cLZ.title" = "Start Speaking";
+"Ynk-f8-cLZ.title" = "Sprechen beginnen";
```

Comment:
Checked the default translation for macOS (which I'm guessing is what these keys are for) - this is apparently "Diktat starten"